contracts/modules/FlashBorrowFees/FlashBorrowFeesHelper.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity 0.5.17;
+
+import "../../utils/MathUtil.sol";
+import "../../core/State.sol";
+import "@openzeppelin-2.5.0/token/ERC20/SafeERC20.sol";
+import "@openzeppelin-2.5.0/token/ERC20/IERC20.sol";
+
+contract FlashBorrowFeesHelper is State{
+	using MathUtil for uint256;
+	function _payFlashBorrowFees(
```

Comment:
there is a special pattern we're following while using modules to register function selectors in proxy state. this function won't be callable if you don't do that

Reply:
add to IBZx.sol interface